doc/sphinx/docs/python/tutorials/using_population.rst

```diff
@@ -115,3 +115,28 @@ some individual decision vector:
           triggering the fitness function evaluation, but it is also possible to inject
           spurious information into the population (i.e. breaking the relation between
           decision vectors and fitness vectors imposed by the problem)
+
+The best individual in a population can be extracted as:
+
+.. doctest::
+
+    >>> # The decision vector
+    >>> pop1.get_x()[pop1.best_idx()]
```

Comment:
agreed, but lets leave it for a syntactic sugars PR in the future ... I guess we may change our minds once we start using pygmo